plugins/jenkins-backend/src/service/jenkinsApi.ts

```diff
@@ -144,32 +137,20 @@ export class JenkinsApiImpl {
    * Trigger a build of a project
    * @see ../../../jenkins/src/api/JenkinsApi.ts#retry
    */
-  async buildProject(
+  async rebuildProject(
     jenkinsInfo: JenkinsInfo,
     jobFullName: string,
-    resourceRef: string,
-    options?: { token?: string },
-  ) {
-    const client = await JenkinsApiImpl.getClient(jenkinsInfo);
-
-    if (this.permissionApi) {
```

Comment:
what's the reason for removing this?

Reply:
I thought it was checking Jenkins token permissions, which would have been duplicative. However, I was completely wrong. I will add it back in. Thanks for reviewing!

Reply:
Thanks for your patience! I added the permission check back in. I also realized that without the SDK I am responsible for handling the crumb issuer for environments that require CSRF. That support is now added.

It looks like there was a hiccup with one of the tests. If you can restart it that would be great, otherwise I can make a tiny commit to restart the tests.